winch/codegen/src/isa/x64/masm.rs

```diff
@@ -39,6 +39,13 @@ use wasmtime_environ::{PtrSize, WasmValType, WASM_PAGE_SIZE};
 pub(crate) struct MacroAssembler {
     /// Stack pointer offset.
     sp_offset: u32,
+    /// This value represents the maximum stack size seen while compiling the function. While the
+    /// function is still being compiled its value will not be valid (the stack will grow and
+    /// shrink as space is reserved and freed during compilstion), but once all instructions have
```

Comment:
```suggestion
    /// shrink as space is reserved and freed during compilation), but once all instructions have
```

Reply:
Amazing spelling mistake, thank you for catching that!